docs/user-documentation/searching.md

```diff
@@ -0,0 +1,60 @@
+# Setup and Configure Search
+
+Islandora 8 comes with the Drupal 8 [Search API](https://www.drupal.org/project/search_api) and [SOLR](https://www.drupal.org/project/search_api_solr) modules enabled with a corresponding SOLR instance. This guide gives an overview to the setup provided by the claw-playbook. Much more detail is available in the [Search API documentation](https://www.drupal.org/docs/8/modules/search-api). Another helpful resource is [Adam Fuch's "Drupal 8 Custom Site Search with Search API"](https://www.electriccitizen.com/blog/drupal-8-custom-site-search-search-api) (https://www.electriccitizen.com, 2018-01-10; last accessed 2019-03-08).
+
+## Indexing Islandora 8 with SOLR
+
+To access the search indexing settings navigate to '/admin/config/search/search-api'.
+
+![Screenshot of the search-api configuration page.](../assets/search-settings-page.png)
+
+### SOLR Server Configuration
+
+Use the 'Solr Server' link to view the SOLR server's configuration and the 'Default Solr content index' link to view or update the index settings. In most cases, were the site was built using ansible, the 'Solr Server' settings were configured during installation should be left alone.
```

Comment:
spelling: In most cases, **were** the site to **where** the site
add word:  'Solr Server' settings **that** were configured